app/schema/api/v4/models.rb

```diff
@@ -394,10 +394,10 @@ def questionnaire
        properties: {
          id: {"$ref" => "#/definitions/uuid"},
          deleted_at: {"$ref" => "#/definitions/nullable_timestamp"},
-         created_at: {"$ref" => "#/definitions/timestamp"},
```

Comment:
We are not syncing this to the client anymore, since the server will take care of syncing just 1 questionnaire per type.